packages/cdktf-cli/lib/get/base.ts

```diff
@@ -48,7 +48,7 @@ export abstract class GetBase {
         });
 
         const pacmak = require.resolve('jsii-pacmak/bin/jsii-pacmak');
-        await shell(pacmak, [ '--target', options.targetLanguage, '--code-only' ]);
+        await shell('node', [ pacmak, '--target', options.targetLanguage, '--code-only' ]);
```

Comment:
According to [this comment](https://github.com/awslabs/cdk8s/pull/183), it might be worthwhile implementing https://github.com/aws/jsii-srcmak rather than fixing this here.

Reply:
I saw you mentioned that elsewhere. I'm not really familiar with what that entails, but I can take a look.

Reply:
see #272